ci/it/testcases/utils.go

```diff
@@ -34,19 +36,53 @@ type Containers struct {
 	ClickHouse    *testcontainers.Container
 }
 
-func (c *Containers) Cleanup(ctx context.Context) {
-	if c.Elasticsearch != nil {
-		(*c.Elasticsearch).Terminate(ctx)
+func printContainerLogs(ctx context.Context, container *testcontainers.Container, name string) {
+	if container == nil {
+		return
 	}
-	if c.Quesma != nil {
-		(*c.Quesma).Terminate(ctx)
+
+	reader, err := (*container).Logs(ctx)
+	if err != nil {
+		log.Printf("Failed to get logs for container '%s': %v", name, err)
+		return
+	}
+
+	output, err := io.ReadAll(reader)
+	if err != nil {
+		log.Printf("Failed to read logs for container '%s': %v", name, err)
+		return
+	}
+
+	log.Printf("Logs for container '%s':", name)
```

Comment:
What if there are a lot of logs, wouldn't it make sense to limit it to certain amount of lines. E.g. last 100 or 1000. I've seen cases in past when logging broke everything.

Reply:
Improved the code to get the last 1MB of logs and 1000 lines from that buffer. This should be safe.